module/VuFind/src/VuFind/Session/AbstractBase.php

```diff
@@ -106,7 +106,7 @@ public function disableWrites()
      *
      * @SuppressWarnings(PHPMD.UnusedFormalParameter)
      */
-    public function open($sess_path, $sess_name)
+    public function open($sess_path, $sess_name): bool
```

Comment:
Correct me if I'm misremembering things, but I suspect that all of these added types are a backward-compatibility break since any class extending these classes will need to be adjusted to match. If that is the price of PHP 8.1 compatibility, it's probably worth it, but we'll definitely want a changelog note summarizing all of the changed classes (which I'm happy to write). Would you agree?

Reply:
True, and I agree.